### DISCUSSION TITLE
LCD flashes when rotary encoder is being used

### DISCUSSION THREAD

**Discussion by murrkywaters** [probonopd/MiniDexed#370] [Q&A]
well, that and there are no characters being displayed. I'm using an i2c backpack, according to a scan the address is 0x3f. Am I missing something?

**Comment by pik33** [comment]
My "backpacked" LCD also didn't show anything at the start. The problem was a small potentiometer on the backpack not properly set. Try to move it and see if it helps.

**Reply by murrkywaters** [reply]
Saw that. It wasn't the issue. I changed which display went to the i2c address and that seemed to fix it.
